solutions/18.js

```diff
@@ -0,0 +1,30 @@
+// Find missing number in array
+// Return the missing number in an input array of consecutively increasing numbers.
+// input: [5,6,8,9]
+// output: 7
+
+// Zach Nagatani
+
+/**
+ * Finds and returns the missing number from an array of consecutively increasing numbers
+ * @param {Number[]} arr - An array of consecutively increasing numbers
+ * @returns {Number} missingNum - the number missing from the input array
+ */
+const solution = arr => {
+    let missingNum = null,
+        i = 0;
+
+    while (!missingNum && i < arr.length - 1) {
```

Comment:
food for thought. Since there is only 1 missing number, and your goal is to return that number, you might not even need the missingNum variable. start `i = 1`.

while (i < arr.length -1 && arr[i-1] +1 === arr[i] )....

You don't have to make this change, just food for thought. Good job!